public/flights/index.html

```diff
@@ -37,6 +36,7 @@
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
 
     <!--========== page-specific files ==========-->
+    <script src="../js/embed.js"></script>
```

Comment:
We don't need this script in this page.

Reply:
I actually moved that from line 14.
Isn't the code for the iframe copy?

Reply:
I think we should remove it actually.

Reply:
It's necessary for the iframe's code copy to clipboard

Reply:
Correctamundo!